src/crimson/osd/recovery_backend.cc

```diff
@@ -136,6 +136,25 @@ seastar::future<> RecoveryBackend::handle_backfill(
   }
 }
 
+seastar::future<> RecoveryBackend::handle_backfill_remove(
+  MOSDPGBackfillRemove& m)
+{
+  logger().debug("{} m.ls=", __func__, m.ls);
```

Comment:
This seems wrong... missing {}?

Reply:
addressed by https://github.com/ceph/ceph/pull/38501/commits/027b3037198b1cf58cbb919091f27ea2d86140bc

Reply:
:+1: